src/pages/docs/step-ca/policies.mdx

```diff
@@ -0,0 +1,295 @@
+ip      |192.168.0.0/24     |192.168.0.1        |10.0.0.1
+email   |@local             |ca@local           |ca@example.com
+uri     |https://*.local    |https://ca.local   |https://ca.example.com
+
+Not every Common Name is a valid DNS domain, IP, email address or URI, however, so that's why it's also possible to provide rules specifically for Common Names.
+Rules for Common Names are evaluated using a strict string comparison.
+
+Type    | Rule              | Matches                               | Non-Matches
+--------|:-:                |:-:                                    |:-:
+cn      |Custom CA Name     |Custom CA Name                         |Different CA Name
+
+
+## SSH Policies
+
+Policies for SSH certificates consist of rules for user and host certificates.
+A policy for SSH user certificates can have rules for the following:
+
+- Email address domains
+- Principals
+
+A policy for SSH host certificates can have rules for the following:
+
+- DNS domains
+- IP ranges and addresses
+- Principals
+
+When an SSH certificate is requested the policy to be evaluated will be based on the certificate type.
+Then the appropriate policy is evaluated for every principal in the certificate.
+
+For SSH host certificates, every principal is matched against `dns` or `ip` rules.
+Evaluation of the `dns` and `ip` rules is the same as for X.509 certificates.
+
+For SSH user certificates, every principal is parsed to check if it's a valid email address and will then be matched against `email` rules.
+Evaluation of `email` rules is performed like those for X.509 certificates.
+If a principal is not a valid email address, it will be matched against `principal` rules.
+Principals are matched using a strict string comparison and it is possible to configure a wildcard to allow any principal name.
+
+If a policy has rules for user certificates and no rule for host certificates, host certificates will be denied.
+The same is true in case there are rules for host certificates and none for user certificates: a user certificate will then always be denied.
+
+
+Type        | Rule              | Matches                               | Non-Matches
+------------|:-:                |:-:                                    |:-:
+dns         |*.local            |host.local                             |host.example.com
+ip          |192.168.0.0/24     |192.168.0.1                            |10.0.0.1
+email       |@devops            |jane@devops                            |john@finance  
+principal   |*                  |johndoe, john, janedoe, jane           |
+principal   |johndoe            |johndoe                                |janedoe                
+
+
+## Policy Configuration
+
+Policies can be administered using the `step` CLI application.
+The commands are part of the [`step ca policy`](https://smallstep.com/docs/step-cli/reference/ca/policy) namespace.
+There are subcommands for administering `authority`, `provisioner` and `acme` policies.
```

Comment:
- Since this doc is for open source, I think we should only mention authority policies outside of the CM call-to-action alert box below. 

- Then, in the CLI help text, I think we should make it clear that `provisioner` and `acme` policies are only available in Certificate Manager. We will get questions from ppl about "how do I turn on provisioner policies" if it's not clear from the help that you can't.

- (Same deal with ACME EAB subcommands?)